mmcv/ops/scatter_points.py

```diff
@@ -25,10 +25,11 @@ def forward(ctx, feats, coors, reduce_type='max'):
                 'mean'. Default: 'max'.
 
         Returns:
-            voxel_feats (torch.Tensor): [M, C]. Reduced features, input
-                features that shares the same voxel coordinates are reduced to
-                one row.
-            voxel_coors (torch.Tensor): [M, ndim]. Voxel coordinates.
+            tuple[torch.Tensor]: A tuple contains two elements. The first one
+            is the reduced features whose shape is [M, C] and which are reduced
+            from input features that share the same voxel coordinates are
+            reduced to one row. The second is voxel coordinates whose shape is
+            [M, ndim].
```

Comment:
tuple[torch.Tensor]: A tuple contains two elements. The first one is the voxel features with shape [M, C] which are respectively reduced from input features that share the same voxel coordinates . The second is voxel coordinates with shape [M, ndim].